src/app/search/search-workflow-table/search-workflow-table.component.scss

```diff
@@ -0,0 +1,10 @@
+@mixin change-mat-icon-size($size: 24px) {
+  font-size: $size;
+  height: $size;
+  width: $size;
+  line-height: $size;
```

Comment:
Are changing the height, width, and line-height necessary?  As opposed to https://stackoverflow.com/a/36562672/4484759

Reply:
The height does seem to be necessary. Otherwise it's very off like this 
![image](https://user-images.githubusercontent.com/16004905/52428071-11a5be00-2ab6-11e9-805f-43939c87d6a6.png)
but you're right that the other ones don't make a difference.

Reply:
You might be able to use flex layout for this. Have the number and star as elements in a div. The div would be fxLayout="row" with maybe  fxLayoutAlign="center center". Might need to experiment with it.